packages/app/next.config.js

```diff
@@ -2,7 +2,14 @@
 // Learn more: https://github.com/expo/expo/blob/master/docs/pages/versions/unversioned/guides/using-nextjs.md#withexpo
 
 const { withExpo } = require('@expo/next-adapter');
-const withTM = require('next-transpile-modules')(['@mf/components']);
+const withFonts = require('next-fonts');
+const withImages = require('next-images');
+const withTM = require('next-transpile-modules');
 const withPlugins = require('next-compose-plugins');
 
```

Comment:
my wallet-connect wrapper also adds some additional items here, but it should be compatible